### PR TITLE
Closes #31: Fix Links and changed Pictures in README

### LINKED ISSUE #31: Adjust README
- Fix links under "Contact"
- Change pictures of Prioritization and Routing
- Change Release Table

### PR DESCRIPTION
**Pull request by jamesrdi**
### Thanks for your PR! Please fill out the following list :)

---

- [ ] I put the ticket or multiple tickets in review
- [x] Commit message format → Closes #&lt;Issue Number&gt; - Your commit message.
- [ ] Sonarcloud link : \<add the link here>
- [x] No documentation update needed
- [ ] Link to PR with documentation update: \<add the link here>
- [x] No Release Notes needed
- [ ] Release Notes :

```

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,15 +211,12 @@ Other combinations might also work but have not been tested.
...
-| 6.3.2   | 2.7.14     |
-| 6.2.2   | 2.7.12     |
-| 6.1.1   | 2.7.10     |
+| 9.0.0   | 3.3.2     |
 
 # 📨Contact
 
-If you have any questions or ideas feel free to create an [issue](https://github.com/kadai-iokadai/issues) or contact us
-via [GitHub Discussions](https://github.com/orgs/kadai-iodiscussions).
+If you have any questions or ideas feel free to create an [issue](https://github.com/kadai-io/kadai/issues) or contact us
```
Comment (CRoberto1926) [inline README.md:211]: There are many more occurrences of this error, I guess:

grafik

**Review by CRoberto1926 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jamesrdi) [reply README.md:211]: You are right, I will change them

File: qa/hooks/commit-msg [hunk 2]
```diff
@@ -4,7 +4,7 @@ COMMIT_MESSAGE="$(cat $MSG_FILE)"
 REGEX='^(Close|Closes|Closed|Fix|Fixes|Fixed|Resolve|Resolves|Resolved) #([0-9]+)'
 if [[ $COMMIT_MESSAGE =~ $REGEX ]]; then
  TICKET=${BASH_REMATCH[2]}
- RESULT=$(curl -s https://api.github.com/repos/kadai-iokadai/issues/$TICKET)
+ RESULT=$(curl -s https://api.github.com/repos/kadai-io/kadai/issues/$TICKET)
```
Comment (CRoberto1926) [inline qa/hooks/commit-msg]: I think that I solved this one already, can you try to rebase your branch on kadai master?

File: README.md [hunk 3]
```diff
@@ -191,35 +191,28 @@ Currently, KADAI provides the following SPIs:
...
 
 # 📚Releases
 
-The list of [releases](https://github.com/kadai-iokadai/releases) contains a detailed changelog.
+The list of [releases](https://github.com/kadai-io/kadai/releases) contains a detailed changelog.
 
 We use [Semantic Versioning](https://semver.org/).
 
 The following compatibility matrix shows the officially supported Spring version for each release.
 Other combinations might also work but have not been tested.
```
Comment (CRoberto1926) [inline README.md]: If you delete the compatibility matrix, then you should also probably delete this sentence, I guess

File: lib/kadai-core-test/src/test/java/acceptance/jobs/AbstractKadaiJobAccTest.java [hunk 4]
```diff
@@ -69,7 +69,7 @@ class AbstractKadaiJobAccTest {
 
   @AfterEach
   void cleanupJobs() {
-    // Dirty Hack, please refactor me with ticket https://github.com/kadai-iokadai/issues/2238
+    // Dirty Hack, please refactor me with ticket https://github.com/kadai-io/kadai/issues/2238
```
Comment (CRoberto1926) [inline lib/kadai-core-test/src/test/java/acceptance/jobs/AbstractKadaiJobAccTest.java]: This link is dead, either we link to the TASKANA issue, or we create a new one in KADAI

**Review by CRoberto1926 — CHANGES_REQUESTED** [review]

File: qa/hooks/commit-msg (hunk 2 above)
Comment (jamesrdi) [reply qa/hooks/commit-msg]: Yes you are right, it is already solved

File: README.md (hunk 3 above)
Comment (jamesrdi) [reply README.md]: Yes you are right

File: lib/kadai-core-test/src/test/java/acceptance/jobs/AbstractKadaiJobAccTest.java (hunk 4 above)
Comment (jamesrdi) [reply lib/kadai-core-test/src/test/java/acceptance/jobs/AbstractKadaiJobAccTest.java]: Yes, the issue has been transferred to KADAI, I will link the new issue

**Review by holgerhagen — APPROVED** [review]

**Review by CRoberto1926 — APPROVED** [review]

**Pull request merged**